storage/innobase/include/univ.i

```
@@ -169,9 +169,6 @@ using the call command. */
 #define UNIV_INLINE static inline
 
 #define UNIV_WORD_SIZE		SIZEOF_SIZE_T
-#if SIZEOF_SIZE_T == 8
-# define HAVE_INNODB_MMAP
-#endif
```

Now we are removing HAVE_INNODB_MMAP, we should also remove from extra/mariabackup/xtrabackup.cc.

---

Good catch. Unfortunately, due to the design of `mariadb-backup`, unknown command line options (such as `innodb_log_file_mmap` in this case) will be silently ignored; I checked it by creating two combinations of the test `mariabackup.innodb_redo_log_overwrite`; we had `log_sys.log_mmap=false` in both, and the test passed. At least the test amended in such a way has a chance of covering both log parsers on all targets.

---

6adffcd96873f57a2b4fd84e51a0829ff8c79f1d extends the test coverage, but it has no way of ensuring that both values of the option are actually covered.